cmd/flow/main.go

```diff
@@ -20,46 +20,126 @@
 package main
 
 import (
+	"github.com/onflow/flow-cli/internal/command"
+	"github.com/onflow/flow-cli/pkg/flow"
+
+	"github.com/onflow/flow-cli/internal/blocks"
+	"github.com/onflow/flow-cli/internal/emulator"
+
```

Comment:
If you're using Goland, I recommend enabling `goimports` to automatically sort and organize import statements: https://www.jetbrains.com/help/go/integration-with-go-tools.html#goimports

Reply:
thank you I will use it